app/models/logbook_entry.rb

```diff
@@ -1,13 +1,13 @@
 class LogbookEntry < ApplicationRecord
   belongs_to :aircraft
-  belongs_to :pilot_in_command, class_name: "User", optional: false
+  belongs_to :pilot_in_command, class_name: "User", optional: false, inverse_of: :logbook_entries
```

Comment:
Does Rails 7 have `inverse_of` by default in `belongs_to`?

Reply:
The inverse_of option in ActiveRecord associations is used to optimize and simplify the behaviour of associations in Rails models. It helps Rails understand the bidirectional relationship between models more explicitly, which can improve performance and ensure the correct functioning of nested attributes and other features. And for some reason, RuboCop asked me to include it in the models.

Reply:
I believe it will try to infer the inverse relationship, but if model names differ (as is the case here) then it may not be able to apply `inverse_of` automatically. This [post](https://stackoverflow.com/a/54112469) explains it well. Maybe `RuboCop` is smart enough to know it would not automatically infer. Anyhow, thanks!

Reply:
Explicitness is always a good thing!